### ISSUE TITLE
Anniversary Date for relationships?

### ISSUE THREAD

**Issue by dnykanen** [monicahq/monica#554]
Include reminder of anniversaries for relationships, whether marriage, first date, commitment, etc.

**Comment by djaiss** [comment]
Yes - the new Contact structure that I've put in place in the last version, actually allows that. We'll add this feature because it makes a lot of sense. I think it'll be part of a new concepts I want to introduce: the notion of **important dates** about someone.

**Comment by bbrendon** [comment]
Life events is added. Should this still be open?

**Comment by Wdavery** [comment]
Looks like this should be closed now.

**Comment by jmorahan** [comment]
Life events have a checkbox to create a reminder, but the reminder isn't actually created unless the life event type has a "name" set in the database, which it seems they don't by default.

**Comment by rorpage** [comment]
Thank you @jmorahan for the tip!!

I ran this (your `Anniversary` event might have a different ID so please check it first):
```
update monica.life_event_types
set name = 'Anniversary'
where id = 11;
```

and now when I create an Anniversary Life Event, a reminder is created! 💯
